src/checkqueue.h

```diff
@@ -26,7 +26,7 @@ class CCheckQueueControl;
   * the master is done adding work, it temporarily joins the worker pool
   * as an N'th worker, until all jobs are done.
   */
-template <typename T>
+template <typename T, size_t J size_t W>
```

Comment:
You're viewing an outdated diff, I just did this for a clean interface change then implementation change set of commits. They are unused here.